Task_3/task_3.py

```diff
@@ -0,0 +1,42 @@
+""" Multi paradigm programming language
+    Task 3
+    Saprykin Maksym
+    zachet_number = 16
+"""
+from string import Template
+import math as m
+import utils as ut
+
+TASK_NUMBER = 'Task 3'
+TEMPLATE = 'input {} = '
+
+
+# ishodna formula
+def calculation_expression(var_x, var_z):
+    return (var_x + 2 * var_x + 3) / (var_z - 2) + m.atan(var_z)
+
+
+# input data
+def inp_validate(values):
+    while True:
+        try:
+            return float(input(TEMPLATE.format(values)))
+        except ValueError:
+            print('Enter not number')
```

Comment:
I noticed you used the same code in the previous lab. could you move this function to utils?